doc/nrf/app_dev/device_guides/wifi_coex.rst

```diff
@@ -269,6 +249,7 @@ The *DT property* is the name of the devicetree node property that configures th
 
 In cases where the GPIO is asserted after the radio activity has begun, the ``GRANT`` signal triggers a software interrupt, which in turn disables the radio.
 No guarantee is made on the latency of this interrupt, but the ISR priority is configurable.
+A priority as close to 0 as possible is recommended to avoid delaying the coex interface.
 
```

Comment:
Should there be a note that using a lead time of 50µs may in rare cases cause the short range radio to be disabled too late? It may be clearer if we mention the COEX2 pin for the nRF91 as a specific use case.

If there is a customer demands in the future for using the 50-100µs anticipation time, it could be added as a KConfig option that uses HW to disable the radio. But that would not work for 802.15.4

Reply:
There is a note here. Do we feel it's not specific enough? https://github.com/nrfconnect/sdk-nrf/pull/16811/files/a1e6032e852291542fe8bb78112cbc5dc0b038e6#diff-faa1a684a544fb79e2ff4e9f9d4035f982a17a16dfa16570f3ce0c787f8d9f45L271